baseapp/baseapp.go

```diff
@@ -34,21 +35,40 @@ type (
 	// Enum mode for app.runTx
 	runTxMode uint8
 
-	// StoreLoader defines a customizable function to control how we load the CommitMultiStore
-	// from disk. This is useful for state migration, when loading a datastore written with
-	// an older version of the software. In particular, if a module changed the substore key name
-	// (or removed a substore) between two versions of the software.
-	StoreLoader func(ms sdk.CommitMultiStore) error
+	// StoreParams is an alias for the config parameter type of the CommitMultiStore.
+	StoreParams = multi.StoreParams
+	// StoreOption provides a functional callback to modify StoreParams.
+	// The callback is passed the loaded height as uint64.
+	// This can be used to control how we load the CommitMultiStore from disk. This is useful for
+	// state migration, when loading a datastore written with an older version of the software.
+	// In particular, if a module changed the substore key name (or removed a substore) between
+	// two versions of the software.
+	StoreOption func(*StoreParams, uint64) error
```

Comment:
so this function will modify `StoreParams`? let's better document that.

Reply:
first line of the godoc